packages/web/src/common/components/Forms/Select.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { useState, useRef, useEffect } from "react";
+import styled, { css } from "styled-components";
+import { KeyboardArrowDown, KeyboardArrowUp } from "@mui/icons-material";
+import Label from "./_atomic/Label";
+import ErrorMessage from "./_atomic/ErrorMessage";
+
+interface SelectItem {
+  label: string;
+  value: string;
+  selectable: boolean;
+}
+
+interface SelectProps {
+  items: SelectItem[];
+  label?: string;
+  errorMessage?: string;
+  disabled?: boolean;
+}
+
+const DropdownContainer = styled.div`
+  gap: 4px;
+  position: relative;
+`;
+
+const disabledStyle = css`
+  background-color: ${({ theme }) => theme.colors.GRAY[100]};
+  border-color: ${({ theme }) => theme.colors.GRAY[200]};
+  pointer-events: none;
+`;
+
+const StyledSelect = styled.div<{
+  hasError?: boolean;
+  disabled?: boolean;
+  isOpen?: boolean;
+}>`
+  width: 300px;
+  padding: 8px 12px;
+  outline: none;
+  cursor: pointer;
+  background-color: ${({ theme }) => theme.colors.WHITE};
+  border: 1px solid
+    ${({ theme, hasError, isOpen }) => {
+      if (isOpen) return theme.colors.PRIMARY;
+      return hasError ? theme.colors.RED[600] : theme.colors.GRAY[200];
+    }};
+  border-radius: 4px;
+  font-family: ${({ theme }) => theme.fonts.FAMILY.PRETENDARD};
+  font-size: 16px;
+  line-height: 20px;
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.REGULAR};
+
+  &:focus,
+  &:hover:not(:focus) {
+    border-color: ${({ theme, isOpen }) =>
+      isOpen ? theme.colors.PRIMARY : theme.colors.GRAY[300]};
+  }
+
+  ${({ disabled }) => disabled && disabledStyle}
+`;
+
+const Dropdown = styled.div`
+  position: absolute;
```

Comment:
요것이 있어야 `gap: 4px;`가 작동(?)한다!!

```suggestion
  position: absolute;
  display: flex;
  flex-direction: column;
```